daemon/api.go

```diff
@@ -778,6 +781,7 @@ func snapInstall(inst *snapInstruction, st *state.State) (string, []*state.TaskS
 }
 
 func snapUpdate(inst *snapInstruction, st *state.State) (string, []*state.TaskSet, error) {
+	// TODO: bail if revision is given (and != current?), *or* behave as with install --revision?
```

Comment:
I don't understand the point made here. If it's already installed, we should block both of them with "already installed", otherwise it's a refresh rather than install.